doc/build-docs.sh

```diff
@@ -11,7 +11,6 @@ fi
 
 # Update git modules if necessary
 git submodule sync
```

Comment:
I think these lines can be removed as well?

Reply:
Then we also don't need the `--recursive` flag anymore when cloning Tribler for experiments (although having it doesn't hurt) 👍

Reply:
Yes, thank you. Removed.